x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -323,20 +338,20 @@ export const getAlertActions = ({
         }
       },
       id: 'addEndpointException',
-      isActionDisabled: () => !canUserCRUD || !hasIndexWrite,
+      isActionDisabled: () => !canUserCRUD || !hasIndexWrite || isEndpointAlert() === false,
```

Comment:
Super nit: I see sometimes we check for `falsiness` using `!` and other times we do `=== false`, is there a specific reason for one over the other throughout?